jscripts/bbcodes_sceditor.js

```diff
@@ -78,46 +78,55 @@ $(document).ready(function($) {
 		format: function($elm, content) {
 			var	fontSize,
 				sizes = ['xx-small', 'x-small', 'small', 'medium', 'large', 'x-large', 'xx-large'],
-				size  = $elm.data('scefontsize');
+				size = '';
+
+			fontSize = $elm.css('fontSize');
+
+			// Most browsers return px value but IE returns 1-7
+			if(fontSize.indexOf('px') > -1) {
+				// convert size to an int
+				fontSize = fontSize.replace('px', '') - 0;
+				size     = 1;
+
+				if(fontSize > 9)
+					size = 2;
+				if(fontSize > 12)
+					size = 3;
+				if(fontSize > 15)
+					size = 4;
+				if(fontSize > 17)
+					size = 5;
+				if(fontSize > 23)
+					size = 6;
+				if(fontSize > 31)
+					size = 7;
+			}
+			else
+				size = (~~fontSize) + 1;
 
-			if(!size)
-			{
-				fontSize = $elm.css('fontSize');
-
-				// Most browsers return px value but IE returns 1-7
-				if(fontSize.indexOf('px') > -1) {
-					// convert size to an int
-					fontSize = fontSize.replace('px', '') - 0;
-					size     = 1;
-
-					if(fontSize > 9)
-						size = 2;
-					if(fontSize > 12)
-						size = 3;
-					if(fontSize > 15)
-						size = 4;
-					if(fontSize > 17)
-						size = 5;
-					if(fontSize > 23)
-						size = 6;
-					if(fontSize > 31)
-						size = 7;
-				}
-				else
-					size = (~~fontSize) + 1;
+			if(size > 7)
+				size = 7;
+			if(size < 1)
+				size = 1;
+
+			size = sizes[size-1];
 
+			return '[size=' + size + ']' + content + '[/size]';
+		},
+		html: function(token, attrs, content) {
+			var sizes = ['xx-small', 'x-small', 'small', 'medium', 'large', 'x-large', 'xx-large'],
+			size = $.inArray(attrs.defaultattr, sizes);
+			if (!isNaN(attrs.defaultattr)) {
+				size = attrs.defaultattr;
 				if(size > 7)
 					size = 7;
 				if(size < 1)
 					size = 1;
-
-				size = sizes[size-1];
 			}
-
-			return '[size=' + size + ']' + content + '[/size]';
-		},
-		html: function(token, attrs, content) {
-			return '<span data-scefontsize="' + attrs.defaultattr + '" style="font-size:' + attrs.defaultattr + '">' + content + '</span>';
+			if (size <= 0) {
+				size = 1;
+			}
+			return '<font data-scefontsize="' + attrs.defaultattr + '" size="' + size + '">' + content + '</font>';
```

Comment:
you need ask to developer of sceditor. Core of plugin use font, so if use span here, issue will happen. 
https://github.com/samclarke/SCEditor/blob/master/src/plugins/bbcode.js#L2283